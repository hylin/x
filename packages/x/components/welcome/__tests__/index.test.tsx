import React from 'react';

import Welcome from '..';
import mountTest from '../../../tests/shared/mountTest';
import rtlTest from '../../../tests/shared/rtlTest';

describe('welcome', () => {
  mountTest(() => <Welcome />);
  rtlTest(() => <Welcome />);
  beforeAll(() => {
    jest.useFakeTimers();
  });

  afterAll(() => {
    jest.useRealTimers();
  });
});
