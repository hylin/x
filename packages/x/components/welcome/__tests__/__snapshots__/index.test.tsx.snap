// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`welcome rtl render component should be rendered correctly in RTL direction 1`] = `
<div
  class="ant-welcome ant-welcome-filled ant-welcome-rtl ant-flex ant-flex-rtl"
>
  <div
    class="ant-welcome-content-wrapper ant-flex ant-flex-align-stretch ant-flex-rtl ant-flex-vertical"
  />
</div>
`;
